sbi/inference/snl/snl.py

```diff
@@ -71,18 +71,25 @@ def __init__(
         # SNL-specific summary_writer fields
         self._summary.update({"mcmc_times": []})
 
-    def __call__(self, num_rounds: int, num_simulations_per_round):
-        """
-        Run SNL over multiple rounds.
+    def __call__(
+        self,
+        num_rounds: int,
+        num_simulations_per_round: Union[List[int], int],
```

Comment:
We will want to make a parametric type `OneOrMore[int]` (e.g., probably another name).